django_cloud_deploy/tests/unit/cli/prompt_test.py

```diff
@@ -286,6 +303,25 @@ def test_existing_prompt(self, unused_mock):
         self.assertEqual(project_id, 'projectid-123')
         self.assertEqual(len(test_io.answers), 1)  # Answer is not used.
 
+    @mock.patch('django_cloud_deploy.cloudlib.project.ProjectClient.'
+                'project_exists',
+                return_value=True)
+    @mock.patch('django_cloud_deploy.cloudlib.project.ProjectClient.'
+                'get_project_permissions',
+                return_value=_FAKE_PERMISSIONS_RESPONSE_EDITOR)
+    def test_existing_prompt_incorrect_permission(self, *unused_mock):
+        test_io = io.TestIO()
+        args = {
+            'backend': 'gae',
+            'project_creation_mode': workflow.ProjectCreationMode.MUST_EXIST,
+            'project_id': 'projectid-123',
+            'use_existing_project': True
+        }
+        test_io.answers.append('projectid-123')
+
+        with self.assertRaises(SystemExit):
```

Comment:
Why SystemExit?

Reply:
if the user passes it as a flag it does a system exit. I forgot to add a test so it just pops out a new answer in the append.